test/test_reason.re

```diff
@@ -146,103 +146,6 @@ let testContext = () => {
   });
 };
 
-let testUseEffect = () => {
```

Comment:
Deleted because already available in OCaml file, right?

Reply:
Yep! Although in a slightly different form.